go.mod

```diff
@@ -17,11 +16,70 @@ require (
 	golang.org/x/net v0.0.0-20210224082022-3d97a244fca7
 	golang.org/x/sys v0.0.0-20210603081109-ebe580a85c40
 	google.golang.org/grpc v1.38.0
-	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 	k8s.io/kubernetes v1.21.0
 	k8s.io/mount-utils v0.21.0
 )
 
+require (
+	github.com/BurntSushi/toml v0.3.1 // indirect
+	github.com/IBM-Cloud/ibm-cloud-cli-sdk v0.6.7 // indirect
+	github.com/IBM/ibmcloud-volume-vpc v1.0.0-beta15 // indirect
```

Comment:
why we need this lib?

Reply:
This lib is used is csi-common which is used by file-csi-driver.
Going forward will remove this